guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,228 @@
+baseurl=http://_server.example.com_/rhel9/BaseOS/
+
+[AppStream]
+name={RepoRHEL9AppStream}
+baseurl=http://_server.example.com_/rhel9/AppStream/
+----
+.. `/etc/yum.repos.d/{project-context}.repo:`
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[{RepoRHEL9ServerSatelliteServerProjectVersion}]
+name={RepoRHEL9ServerSatelliteServerProjectVersion}
+baseurl=http://_server.example.com_/sat6/Satellite/
+
+[{RepoRHEL9ServerSatelliteMaintenanceProjectVersion}]
+name={RepoRHEL9ServerSatelliteMaintenanceProjectVersion}
+baseurl=http://_server.example.com_/sat6/Maintenance/
+----
+endif::[]
+endif::[]
+
+ifdef::foreman-el,katello[]
+. Install additional OS specific packages (`leapp-data-almalinux` for AlmaLinux, `leapp-data-centos` for CentOS Stream, or `leapp-data-rocky` for Rocky Linux).
+Note that this is not required for {RHEL} based installations.
++
+----
+# {project-package-install} leapp-data-centos
+----
++
+. Add {Project} specific repositories to `/etc/leapp/files/leapp_upgrade_repositories.repo`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[leapp-foreman]
+name=Foreman {ProjectVersion}
+baseurl=https://yum.theforeman.org/releases/{ProjectVersion}/el9/$basearch
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+enabled=1
+gpgcheck=1
+
+[leapp-foreman-plugins]
+name=Foreman plugins {ProjectVersion}
+baseurl=https://yum.theforeman.org/plugins/{ProjectVersion}/el9/$basearch
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+
+leapp-puppet7]
+name=Puppet 7 Repository el 9 - $basearch
+baseurl=http://yum.puppetlabs.com/puppet7/el/8/$basearch
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-2025-04-06-puppet7-release
+enabled=1
+gpgcheck=1
+
+ifdef::katello[]
+[leapp-katello]
+name=Katello {KatelloVersion}
+baseurl=https://yum.theforeman.org/katello/{KatelloVersion}/katello/el9/$basearch/
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+enabled=1
+gpgcheck=1
+
+[leapp-katello-candlepin]
+name=Candlepin: an open source entitlement management system.
+baseurl=https://yum.theforeman.org/katello/{KatelloVersion}/candlepin/el9/$basearch/
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+enabled=1
+gpgcheck=1
+
+[leapp-pulpcore]
+name=pulpcore: Fetch, Upload, Organize, and Distribute Software Packages.
+baseurl=https://yum.theforeman.org/pulpcore/{PulpcoreVersion}/el9/$basearch/
+gpgkey=https://yum.theforeman.org/pulpcore/{PulpcoreVersion}/GPG-RPM-KEY-pulpcore
+enabled=1
+gpgcheck=1
+endif::[]
+----
+
+endif::[]
+. Let Leapp analyze your system:
++
+----
+# leapp preupgrade
+----
+ifdef::satellite[]
++
+ifeval::["{mode}" == "disconnected"]
+Add the `--no-rhsm` and `--enablerepo` parameters:
+endif::[]
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# leapp preupgrade \
+--no-rhsm \
+--enablerepo BaseOS \
+--enablerepo AppStream \
+--enablerepo {RepoRHEL9ServerSatelliteServerProjectVersion} \
+--enablerepo {RepoRHEL9ServerSatelliteMaintenanceProjectVersion}
+----
+endif::[]
```

Comment:
This currently renders wrongly for Satellite.

disconnected:
![image](https://github.com/theforeman/foreman-documentation/assets/94284/cd432d67-ddc7-4d0f-89cb-338fe1658af5)

connected:
![image](https://github.com/theforeman/foreman-documentation/assets/94284/08be71a4-45a1-4472-91d6-d605bad16c3b)

I think we should:
* for connected only show the `leapp preupgrade` (exactly like Katello/Foreman)
* for disconnected ONLY show `leapp preupgrade <many options>` (and avoid the "Add the … parameters")